invenio_records_rest/views.py

```diff
@@ -64,42 +64,39 @@ def elasticsearch_query_parsing_exception_handler(error):
     return InvalidQueryRESTError(description=description).get_response()
 
 
-elasticsearch_error_handlers = {
```

Comment:
My plan was actually to do the `obj_or_import_string` call during app initialization, so that if for some reason the config is wrong or the import fails, the application won't start at all, instead of failing when the time comes for the specific error code to be handled.